### DISCUSSION TITLE
🗳️ 2025 TSC Elections!  🗳️

### DISCUSSION THREAD

**Discussion by leebyron** [graphql/graphql-wg#1611] [Announcements]
Tt's time to nominate and vote for TSC members for the upcoming 2025 year!

If you are interested in joining the TSC, please fill out this form https://forms.gle/czL7BmtNASM13DfP9

Please see this Issue where full details are listed: https://github.com/graphql/graphql-wg/issues/1612

**Comment by leebyron** [comment]
cc @graphql/tsc for awareness!

**Reply by JoviDeCroock** [reply]
@leebyron The form sais the vote closes in 2023, might confuse some people

**Reply by leebyron** [reply]
Ah yikes - fixed late but fixed none the less
